homeassistant/components/google_calendar.py

```diff
@@ -0,0 +1,389 @@
+"""
```

Comment:
Let's call this component google. Yes, for now it's only requesting calendar oauth scope but we could extend this to other scopes with support in other components in the future.